### ISSUE TITLE
SWARM-REGISTER: autonomous worker

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1467]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
INSTANCE_TAG: SOL-SCI-CLOSER-20260912T0606+0200-REAL-LEARNED-MODEL
STATUS: REGISTERED_READ_ONLY_SCOUT_COMPLETE_NONOWNING_PREQUAL_SELECTED
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_REGISTRATION: f250f24fd0004c5021ecbc637ab17d959c266e1d
CURRENT_INTEGRATION_OWNER: #1363

MISSION:
Minimize TIME_TO_REAL_LEARNED_MODEL and SCIENTIFIC_DISTANCE_TO_LEARNED_MODEL. Start genuine bounded LOCAL_FREE learned-20M training immediately only when binding optimized-target exposure is positive and lawful runtime/launch authority is terminal. Until then consume the nearest lawful execution/repair/audit/integration gate; no fake training/weights, no paid compute, no foreign pretrained weights, no external LLM/API corpus intelligence, no final-test leakage.

READ_ONLY LIVE SCOUT COMPLETE BEFORE THIS WRITE:
- reread current main, #723, #548, PROJECT_INSTRUCTIONS, AUTOPULSE_CONTROL, CI_SWARM_POLICY, UNIVERSAL_SWARM_PROMPT and swarm300_protocol_v2;
- current main exact f250f24fd0004c5021ecbc637ab17d959c266e1d;
- #548 is a control checkpoint, not training authorization; it requires positive immutable post-pack training-authorized exposure before optimizer step 1;
- AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0; TOKENIZER_FIT_AUTHORIZED=false; optimizer updates on real targets=0; TRAINING_EXECUTED=false; LEARNED_WEIGHTS_CREATED=false;
- current survivor authority repeatedly reported by live integration control is 5,921,963 meaningful positions vs binding 10,000,000 floor, deficit >=4,078,037;
- Rada PR #655 real LOCAL_FREE Q/P run 34669418910 is real SUCCESS: 101,559 accepted chunks / 192,078,166 observed-not-credited bytes, but exact final head 43a7d62854255da4426d6a3dc4d5ad9b1e4898e9 still has zero check-runs; #1441 owns Product closeout and #1451 released preaudit pending legitimate exact-head shared CI;
- Rada post-Q/P incumbent global-dedup intake is actively owned by #1448; do not duplicate;
- runtime PR #1336 exact head b57ebfb1c4ade9db720c3fbbecc07363125b5c9c is unchanged and exact-head CI 34669470070 remains FAILURE on locked-x86-64; same-lineage Product owner #1440 owns repair;
- D04 PR #1410 exact head a461345d7bffe2d7e93fa2a3e2c1f53f76df0a75 remains the terminal CHANGES_REQUIRED_HIGH head; #1405 owns repair;
- ArXiv+LangUK PR #1404 exact head a0a6060c7fb0bb6358f785b5650050117a6a8fe0 remains terminal CHANGES_REQUIRED_HIGH; #1397 owns repair;
- integration #1363 remains active and owns merges/intake.

CANDIDATE SET BEFORE REGISTRATION:
1. PR #655 exact-final-head fresh audit — highest immediate value, but BLOCKED because exact head still has zero legitimate shared-CI check-runs.
2. PR #1336 fresh changed-head audit — BLOCKED because head has not moved and exact-head CI is terminal FAILURE.
3. PR #1410 / #1404 changed-head fresh audit — BLOCKED because producers have not moved past terminal HIGH-rejected heads.
4. Duplicate #1448 Rada downstream execution — REJECT due active Product ownership.
5. Start learned-20M optimizer training now — REJECT as scientifically unauthorized while optimized-target exposure=0 and tokenizer fit=false.
6. SELECTED FALLBACK: disjoint NON-OWNING exact prequalification/diagnostic on runtime PR #1336 to find any launch-critical defect beyond the already-known Ruff blocker, then hand concrete reproducible evidence to lawful owner #1440. No Product mutation, no authoritative audit verdict, no merge.

HARD BOUNDARY:
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
FINAL_TEST_OUTCOMES_READ=false
PRODUCT_MUTATION=false
AUTHORITATIVE_AUDIT_VERDICT=false
MERGE_AUTHORITY=false

NEXT:
Derive deterministic routing from this issue number, inspect exact PR #1336 changed files and live runtime authority. If a concrete defect is found, publish a non-owning handoff to #1440. If a fresh terminal-CI critical head appears first, rotate immediately and claim/audit only after mandatory direct OPEN arbitration. If optimized-target authority turns positive and runtime becomes terminal launchable, switch immediately to real bounded LOCAL_FREE training unless another worker already owns the exact run.

**Comment by Oleksii-debug** [comment]
ROUTING + DELIVERED ACTION

SWARM_WORKER_ID: SWARM-1467
PREFERRED_LANE: #14 (2 + 1467 mod 15)
PREFERRED_WORK_KIND: INDEPENDENT_VERIFY (work_kinds[(1467 div 15) mod 8])
SELECTED_MODE: NON-OWNING exact-head prequalification because all higher-value terminal audit/execution transitions were blocked or live-owned.

DELIVERED: `PREQUAL1467-D02-009 — HIGH` on PR #1336 exact head `b57ebfb1c4ade9db720c3fbbecc07363125b5c9c`.
- durable PR comment: 5643378140
- owner #1440 handoff: 5643378756
- finding: live model parameter tensor/start-state is not authenticated against independently trusted FRESH_START random-init authority or RESUME parent-checkpoint authority; ModelSpec/InitSpec metadata + parameter count can front arbitrary same-shaped mutated/loaded weights.
- required same-lineage regression: mutate/load same-shaped weights with unchanged metadata and prove BLOCKED_PRE_STEP_1 before optimizer/counter/D04 mutation.

NO Product mutation, no audit verdict, no merge, no training claim. Refreshing live heads/CI immediately for next lawful transition.

**Comment by Oleksii-debug** [comment]
CLAIM — SWARM-1467

STATUS: ACTIVE_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: AUDIT-B|PR1336|START-STATE-AUTHENTICATION|EXACTHEAD-B57EBFB1|V1
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / EXECUTABLE_ADVERSARIAL_REGRESSION
TARGET_PRODUCT_PR: #1336
TARGET_PRODUCT_HEAD: b57ebfb1c4ade9db720c3fbbecc07363125b5c9c
PRODUCT_OWNER: #1440
CURRENT_INTEGRATION_OWNER: #1363
LOCAL_FREE_ONLY: true

OBJECTIVE:
Turn existing `PREQUAL1467-D02-009` HIGH into one executable independent verifier without mutating Product code: prove that same-shaped live parameter-state substitution with unchanged ModelSpec/InitSpec metadata must fail closed before optimizer step 1 / Trainer target counters / D04 exposure mutation.

OWNED SURFACE IF POSTCLAIM ARBITRATION WINS:
- exactly one additive verifier test file on a new audit branch rooted at exact Product head b57;
- no Product implementation mutation;
- no #1440 branch mutation;
- no D04/readiness/merge/training authority;
- at most one verifier PR for this claim.

ACCEPTANCE:
1. Mandatory direct OPEN issue and PR collections reach exact empty sentinels and expose no earlier live same-semantic start-state verifier/auditor.
2. Product remains exact b57 immediately before branch creation; head movement invalidates this claim target and forces refresh.
3. Regression starts from otherwise-valid bounded-pilot fixture, mutates/loads same-shaped parameter values while retaining ModelSpec/InitSpec metadata, then requires fail-closed before optimizer/counters/D04 mutation.
4. Verifier itself does not claim Product PASS; expected red against vulnerable b57 is admissible evidence. Any inherited Product Ruff failure is reported truthfully rather than mislabeling tests executed.
5. Release verifier ownership after durable PR/handoff so future repaired Product head can rebase/rerun under fresh arbitration.

HARD BOUNDARY:
PRODUCT_MUTATION=false
MERGE_AUTHORITY=false
TRAINING_AUTHORITY=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
FINAL_TEST_OUTCOMES_READ=false

NEXT: mandatory postclaim direct OPEN issue+PR pagination to exact empty sentinels before branch/file/PR creation.

**Comment by Oleksii-debug** [comment]
ARBITRATION WIN — SWARM-1467

CLAIM_KEY: `AUDIT-B|PR1336|START-STATE-AUTHENTICATION|EXACTHEAD-B57EBFB1|V1`
TARGET_PRODUCT_PR: #1336
TARGET_PRODUCT_HEAD_REBOUND: `b57ebfb1c4ade9db720c3fbbecc07363125b5c9c`
STATUS: ACTIVE_ARBITRATION_WIN_EXECUTABLE_VERIFIER

Mandatory postclaim direct arbitration completed before any branch/file write:
- OPEN issues: direct REST newest-first, `per_page=100`, pages 1..10 nonempty, page 11 exact `[]` sentinel.
- OPEN PRs: direct REST newest-first, `per_page=100`, pages 1..6 nonempty, page 7 exact `[]` sentinel.
- targeted open semantic search for `START-STATE-AUTHENTICATION` returns only SWARM-1467.
- broader search for `live model-weight` / `random-init metadata` / `same-shaped weights` found permanent AUDIT-B, Product owner #1440, and SWARM-1467; no earlier live same-semantic verifier/Product owner.
- existing PR #1465 is different semantic: AdamW behavior flags + actual scheduler-object authority under #1461, explicitly not this live start-state/weight provenance gap.
- exact Product #1336 rebound after arbitration: OPEN, non-draft, mergeable, head still exactly `b57ebfb1c4ade9db720c3fbbecc07363125b5c9c`.

ARBITRATION RESULT: WIN for one additive independent verifier only.

AUTHORIZED MUTATION SURFACE:
- create one audit branch rooted exactly at b57;
- add exactly one verifier test file proving same-shaped live parameter substitution cannot pass as canonical random-init/resume state;
- open at most one verifier PR stacked on Product branch;
- no modification of `src/twelve_six/training/bounded_pilot.py`, Product tests, #1440 branch, D04/readiness/corpus code, workflows, or main;
- no Product PASS, merge authority, corpus/training credit, or learned-weight claim.

NEXT: inspect exact b57 positive-path fixture/API, create additive adversarial regression, then publish verifier PR/handoff. Expected-red against vulnerable b57 is valid evidence; inherited Product Ruff failure is not mislabeled as execution of the new regression.
